FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m

```diff
@@ -456,4 +459,23 @@ - (BOOL)isDefaultApp {
   return [self.appName isEqualToString:kFIRDefaultAppName];
 }
 
+#pragma mark - Keychain
```

Comment:
out of curiosity?
what does the hash (#) prefix do?

Reply:
[This](https://nshipster.com/pragma/) looks like a good summary.